packages/Popover/src/Popover.js

```diff
@@ -0,0 +1,500 @@
+
+        case 'right':
+          return referenceElPos.left + referenceElPos.width;
+      }
+    }
+
+    if (justification) {
+      switch (justification) {
+        case 'left':
+          return referenceElPos.left;
+
+        case 'right':
+          return (
+            referenceElPos.left + referenceElPos.width - contentRefPos.width
+          );
+
+        case 'center-horizontal':
+          return (
+            referenceElPos.left +
+            referenceElPos.width / 2 -
+            contentRefPos.width / 2
+          );
+      }
+    }
+  }
+
+  calcWithoutPortalLeft({ alignment, justification }) {
+    const contentRefPos = this.getRefPosition(this.contentRef);
+    
+    if (contentRefPos && alignment) {
+      switch (alignment) {
+        case 'left':
+          return `${0 - contentRefPos.width}px`;
+
+        case 'right':
+          return '100%';
+      }
+    }
+
+    if (justification) {
+      switch (justification) {
+        case 'left':
+          return 0;
+
+        case 'right':
+          return `calc(100% - ${contentRefPos.width}px)`;
+
+        case 'center-horizontal':
+          return `calc(50% - ${contentRefPos.width}px / 2)`;
+      }
+    }
+  }
+
+  ////////////////////////////////////////////////////////////////////////////////////////////
+
+  // Get transform styles for position object
+  getTransform(alignment) {
+    const transformAmount = 12;
+    const scaleAmount = 0.8;
+
+    switch (alignment) {
+      case 'top':
+        return `translate3d(0, ${transformAmount}px, 0) scale(${scaleAmount})`;
+
+      case 'bottom':
+        return `translate3d(0, -${transformAmount}px, 0) scale(${scaleAmount})`;
+
+      case 'left':
+        return `translate3d(${transformAmount}px, 0, 0) scale(${scaleAmount})`;
+
+      case 'right':
+        return `translate3d(-${transformAmount}px, 0, 0) scale(${scaleAmount})`;
+    }
+  }
+
+  // Constructs the transform origin for any given pair of alignment / justification
+  getTransformOrigin({ alignment, justification }) {
+    let x, y;
+
+    switch (alignment) {
+      case 'left':
+        x = 'right';
+        break;
+
+      case 'right':
+        x = 'left';
+        break;
+
+      case 'bottom':
+        y = 'bottom';
+        break;
+
+      case 'top':
+        y = 'top';
+        break;
+    }
+
+    switch (justification) {
+      case 'left':
+        x = 'left';
+        break;
+
+      case 'right':
+        x = 'right';
+        break;
+
+      case 'bottom':
+        y = 'top';
+        break;
+
+      case 'top':
+        y = 'bottom';
+        break;
+
+      case 'center-horizontal':
+        x = 'center';
+        break;
+
+      case 'center-vertical':
+        y = 'center';
+        break;
+    }
+
+    return `${x} ${y}`;
+  }
+
+  ////////////////////////////////////////////////////////////////////////////////////////////
+
+  render() {
+    const {
+      children,
+      isActive,
```

Comment:
Been thinking a bit about this a bit, and I _think_ that calling this `active` would be better.

Reply:
👍